golem_cli/src/command/yagna.rs

```diff
@@ -10,22 +12,81 @@ use std::process::Stdio;
 use tokio::process::{Child, Command};
 use ya_core_model::payment::local::{InvoiceStats, InvoiceStatusNotes, StatusNotes, StatusResult};
 
+pub static DEFAULT_DRIVER: &'static str = "zksync";
+pub static DEFAULT_NETWORK: &'static str = "mainnet";
+
 pub struct PaymentType {
     pub platform: &'static str,
+    pub token: &'static str,
     pub driver: &'static str,
+    pub token_display_name: &'static str,
 }
 
-impl PaymentType {
-    pub const ZK: PaymentType = PaymentType {
-        platform: "ZK-NGNT",
-        driver: "zksync",
+pub struct DriverDescriptor(pub HashMap<&'static str, PaymentType>);
+
+lazy_static! {
+    pub static ref ZKSYNC_DRIVER: DriverDescriptor = {
```

Comment:
FYI: These information will be obtainable via CLI in the future (not yet tough).